spec/models/spree/order_spec.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: false
+
```

Comment:
Thanks for bringing this back. Ideally, you would have removed this detour from the git history. When I saw this removed in a previous commit, I then had to skip ahead and check if you brought it back or if I needed to comment on it.

Reply:
Yeah, something must have automatically removed those lines so I had to put it back. But I get your point! Removed from the git history.